src/elements/content-sidebar/activity-feed/task-form/TaskForm.js

```diff
@@ -62,16 +70,20 @@ type State = {|
 class TaskForm extends React.Component<Props, State> {
     static defaultProps = {
         approverSelectorContacts: [],
+        dueDate: null,
+        editMode: TASK_EDIT_MODE_CREATE,
```

Comment:
nit: Any reason the mode is not a boolean ? We can get rid of the constants.

Reply:
I tend to this of props as adjectives, describing a characteristic. In this case, a boolean would be more like a transient state) the current logic *could* be boolean, but I find a discrete value to carry more meaning and lend itself to extension if new modes arise

Reply:
It seems like detecting whether the item has an _id_ is sufficient to tell if it should be edited or created

Reply:
@DanDeMicco We can revisit this as I am adding `id` as part of CRUD work.